README.md

```diff
@@ -92,6 +92,19 @@ https://docs.google.com/spreadsheets/d/DOCUMENT_ID/edit#gid=0
 Here, "DOCUMENT_ID" will be a long string of characters, letters, and numbers. 
 This is the unique identifier for the Google Sheet.
 
+#### name (string)
+This parameter allows you to specify a name for the google sheet that should be read and/or altered. \
+This is the name of the tab on the bottom left that you want to read and/or alter
```

Comment:
Write "sync" instead of "read and/or alter" to make it shorter and to keep the word "sync" everywhere.